Jenkinsfile

```diff
@@ -38,10 +38,10 @@ pipeline {
     }
 
     stages {
-        stage('Check Whitelist') {
+        stage('Check User Authorization') {
             steps {
-                readTrusted 'bin/whitelist'
-                sh './bin/whitelist "$CHANGE_AUTHOR" /etc/jenkins-authorized-builders'
+                readTrusted 'bin/authorize-cicd'
+                sh './bin/authorize-cicd "$CHANGE_AUTHOR" /etc/jenkins-authorized-builders'
```

Comment:
How about 'check-authlist' as a script name?

Reply:
That's also good. I prefer `authorize-cicd`, though, because when you are looking in the bin directory it is explicit about what is being authorized (cicd) while using the same verb-object format and character count.